app/components/authorCV/educationForm.tsx

```diff
@@ -198,29 +163,17 @@ class EducationForm extends React.PureComponent<EducationFormProps> {
                         <div className={styles.formInputBox}>
                           <div className={styles.dateInputWrapper}>
                             <Field
-                              name="endDateMonth"
-                              component={scinapseFormikSelect}
-                              placeHolderContent="Month"
-                              defaultValue={values.endDateMonth}
-                              inputStyle={inputStyle}
-                              children={monthItems}
-                              className={classNames({
-                                [styles.dateMonthField]: true,
-                                [styles.errorInputField]: !!errors.endDateMonth,
-                              })}
-                            />
-                            <Field
-                              name="endDateYear"
+                              name="endDate"
                               type="text"
                               placeholder="Year"
                               style={{ color: "#666d7c" }}
                               maxLength="4"
                               className={classNames({
                                 [styles.dateYearField]: true,
-                                [styles.errorInputField]: !!errors.endDateMonth,
+                                [styles.errorInputField]: !!errors.endDate,
                               })}
                             />
-                            <ErrorMessage name="endDateMonth" className={styles.errorMessage} component="div" />
+                            {/* <ErrorMessage name="endDate" className={styles.errorMessage} component="div" /> */}
```

Comment:
If you don't need this line, remove ErrorMessage component. otherwise, uncomment this line, please.